scss/main.scss

```diff
@@ -287,23 +287,48 @@ h1,h2,h3 {
   margin-bottom: 2rem;
   padding-left: 2rem;
   padding-right: 2rem;
+  p.text-center {
+    margin-bottom: 48px;
+  }
 }
 
 #meetupLink {
   color: $cfa-blue;
 }
 
+.meetup-content {
+  margin-bottom: 1.5rem;
+  > * {
+  margin-bottom: 1rem;
+  }
+}
+
+
+
 .rsvp-stuff {
   margin-top: 1rem;
+  p {
+    font-family: $font-family-alt;
+    font-weight: 500;
+    color: $darker-grey;
+    font-size: 14px; 
```

Comment:
Instead of hardcoded pixels, let's use REM instead, in case the user uses a larger default font.
14px equates to 0.875rem
Please adjust in `attend-button` as well.

Thanks!